.github/pull_request_template.md

```diff
@@ -1,25 +1,14 @@
 <!-- if needed please write above the given line -->
+### Thanks for your PR! Please fill out the following list :)
+
 ---
-Release Notes:
+- [ ] I put the ticket in review
```

Comment:
Maybe we should explicitly note that there might be two tickets?

Reply:
That's a great idea!